src/System.Net.Requests/tests/HttpWebRequestTest.cs

```diff
@@ -1098,18 +1106,24 @@ public void BeginGetRequestStream_CreatePostRequestThenCallTwice_ThrowsInvalidOp
             });
         }
 
-        [Theory, MemberData(nameof(EchoServers))]
-        public void BeginGetRequestStream_CreateRequestThenBeginGetResponsePrior_ThrowsProtocolViolationException(Uri remoteServer)
+        [Fact]
+        public async Task BeginGetRequestStream_CreateRequestThenBeginGetResponsePrior_ThrowsProtocolViolationException()
         {
-            HttpWebRequest request = HttpWebRequest.CreateHttp(remoteServer);
-
-            IAsyncResult asyncResult = request.BeginGetResponse(null, null);
-            Assert.Throws<ProtocolViolationException>(() =>
+            await LoopbackServer.CreateServerAsync((server, url) =>
             {
-                request.BeginGetRequestStream(null, null);
+                HttpWebRequest request = HttpWebRequest.CreateHttp(url);
+
+                IAsyncResult asyncResult = request.BeginGetResponse(null, null);
+                Assert.Throws<ProtocolViolationException>(() =>
+                {
+                    request.BeginGetRequestStream(null, null);
+                });
+
+                return Task.CompletedTask;
             });
         }
 
+        [Fact]
```

Comment:
I wonder how many pther tests we have across the repo that are accidentally missing [Fact] or [Theory].  Can we run an analyzer to find other such cases?

Reply:
Opened https://github.com/dotnet/corefx/issues/38022